_1327/minutes/models.py

```diff
@@ -68,7 +68,7 @@ class Meta(Document.Meta):
 
 	@classmethod
 	def generate_new_title(cls):
-		return _("Minutes")
+		return "Minutes", "Minutes"
```

Comment:
It's true that I forgot the translation here.
The problem with the translation is, that it is based on the current lanugage but I need this function to return the English and the German title. If the selected language is German then `return "Minutes", _("Minutes")` would work but if English is selected `("Minutes","Minutes")` would be returned.
For now I thought it would be the best way to hardcode these values because they should be changed anyways.

Reply:
true. there are probably ways to explicitly set the language for that one translation. that would probably be the cleanest solution, but i don't know whether it's worth it.